src/FBAEMReporter.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+import {NativeModules, Platform} from 'react-native';
+
+/**
+ * Aggregated Event Measurement (AEM) for iOS apps allows for the measurement of app events
+ * from iOS 14.5+ users who have opted out of app tracking.
+ *
+ * @ref https://developers.facebook.com/docs/app-events/guides/aggregated-event-measurement/
+ */
+export default {
+  /**
+   * Log AEM event, event names for AEM must match event names you used in app event logging.
+   *
+   * *This method will bypass if platform isn't iOS*
+   *
+   * **Make sure you also handle the DeepLink URL with AEM**
+   *
+   * @ref https://developers.facebook.com/docs/app-events/guides/aggregated-event-measurement/
+   * @platform iOS
+   */
+  logAEMEvent: (
```

Comment:
Reviewing from mobile, but isn't it only iOS available?
In case it won't be ported to android in near future, I would make it platform specific as we did for other APIs instead of fallback in android, just for sake of consistency.

In case we do have an android version of it and it's just a case of porting it I would make it clear that support for Android would come soon and/or we're open for contributors :)

Reply:
Yes, it's iOS only feature. I'm not sure what you mean by "make it platform specific instead of fallback in android", I thought marking it as `@platform iOS` is what we do for platform specific, the actual use case, as far as I know, is to send an additional event using logAEMEvent

```js
  AppEventsLogger.logEvent(eventName, value, parameters);
  AEMReporter.logAEMEvent(eventName, value, currency, parameters);
```
The early return on the none-ios platform is to make the function usage easier, so we don't have to do something as below
```js
  AppEventsLogger.logEvent(eventName, value, parameters);
  if(Platform.OS === 'ios'){
    AEMReporter.logAEMEvent(eventName, value, currency, parameters);
  }
```
I don't know if there will be an android version of AEM, not mentioned on the document, but I think it will not be happening in the near future.

Reply:
I got your point, problem is that someone implementing it could be confused and expect that it would works for android as well.

https://developers.facebook.com/docs/facebook-login/limited-login/ios

Take a look in the above part in our Readme.md

Reply:
Sorry I meant this method: loginTrackingIOS

NB!: the iOS app doesn't help in copy/paste some internal links :(

Reply:
Taking your advice, I renamed it to `AEMReporterIOS.logAEMEvent`